flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -529,6 +529,24 @@ public void testDropIfExists() {
 		check(sql, "DROP TABLE IF EXISTS `CATALOG1`.`DB1`.`TBL1`");
 	}
 
+	@Test
+	public void testCreateFunction() {
```

Comment:
add a test for "CREATE FUNCTION XXX IF NOT EXIST"?

Reply:
Added.